store/tikv/rawkv_test.go

```diff
@@ -89,13 +122,10 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 }
 
 func (s *testRawKVSuite) TestSplit(c *C) {
-	loc, err := s.client.regionCache.LocateKey(s.bo, []byte("k"))
-	c.Assert(err, IsNil)
 	s.mustPut(c, []byte("k1"), []byte("v1"))
 	s.mustPut(c, []byte("k3"), []byte("v3"))
 
-	newRegionID, peerID := s.cluster.AllocID(), s.cluster.AllocID()
-	s.cluster.SplitRaw(loc.Region.id, newRegionID, []byte("k2"), []uint64{peerID}, peerID)
+	s.split(c, "k", "k2")
```

Comment:
Let `split` return error and `c.Assert(s.split(c, "k", "k2"), IsNil)` here,
or rename split to `mustSplit`

I prefer former, because when there are many `split()` call and assert failed, we won't lost the error stack.